selfdrive/car/card.py

```diff
@@ -245,7 +249,7 @@ def state_publish(self, CS: car.CarState, RD: structs.RadarDataT | None):
       tracks_msg.liveTracks = RD
       self.pm.send('liveTracks', tracks_msg)
 
-  def controls_update(self, CS: car.CarState, CC: car.CarControl):
+  def controls_update(self, CS: car.CarState, CC: car.CarControl, CC_SP: custom.CarControlSP):
```

Comment:
```suggestion
  def controls_update(self, CS: car.CarState, CC: car.CarControl, CarControlSP: custom.CarControlSP):
```

We don't get charged by words, we shouldn't need to try to stick to the same confusing naming comma has